oracledb-ballerina/Dependencies.toml

```diff
@@ -0,0 +1,10 @@
+[[dependency]]
+org = "ballerina"
+name = "sql"
+version = "@stdlib.sql.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "crypto"
+version = "@stdlib.crypto.version@"
+
```

Comment:
```suggestion
```

Reply:
Should I remove the last new line of all .toml files?

Reply:
we need to keep one new line at the end of the file. seems like this has two.

Reply:
Resolved with 171baaf97be39b48c6663ec8edeb3d05944f98f0